sdk/core/azure-core/inc/azure/core/http/body_stream.hpp

```diff
@@ -37,6 +37,20 @@ namespace Azure { namespace Core { namespace Http {
    *@brief Used to read data to/from a service.
    */
   class BodyStream {
+  protected:
```

Comment:
Do we need this to be protected, or can it remain private?
```suggestion
  private:
```